### ISSUE TITLE
[matplotplusplus] build failure

### ISSUE THREAD

**Issue by Bzz2745** [microsoft/vcpkg#32253]
### Operating system

Windows 11

### Compiler

MSVC vc143,CUDA 12.1

### Steps to reproduce the behavior

```Shell
`./vcpkg install matplotplusplus[core,blas,fftw,jpeg,lapack,opencv,opengl,zlib]:x64-windows --clean-after-build`
```

### Failure logs

Installing 1/1 matplotplusplus:x64-windows...
Building matplotplusplus[blas,core,fftw,jpeg,lapack,opencv,opengl,zlib]:x64-windows...
--  matplotplusplus's gnuplot backend currently requires Gnuplot 5.2.6+.
    Windows users may get a pre-built binary installer from http://www.gnuplot.info/download.html.
    Linux and MacOS users may install it from the system package manager.
    Please visit https://alandefreitas.github.io/matplotplusplus/ for more information.
-- Note: matplotplusplus only supports static library linkage. Building static library.
-- Using cached alandefreitas-matplotplusplus-v1.2.0.tar.gz.
-- Extracting source C:/vcpkg/downloads/alandefreitas-matplotplusplus-v1.2.0.tar.gz
-- Applying patch fix-dependencies.patch
-- Using source at C:/vcpkg/buildtrees/matplotplusplus/src/v1.2.0-b6c461fd66.clean
-- Configuring x64-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: C:/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe -v
    Working Directory: C:/vcpkg/buildtrees/matplotplusplus/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\vcpkg\buildtrees\matplotplusplus\config-x64-windows-dbg-CMakeCache.txt.log
      C:\vcpkg\buildtrees\matplotplusplus\config-x64-windows-rel-CMakeCache.txt.log
      C:\vcpkg\buildtrees\matplotplusplus\config-x64-windows-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:252 (vcpkg_execute_required_process)
  ports/matplotplusplus/portfile.cmake:32 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)

### Additional context

[config-x64-windows-dbg-CMakeCache.txt.log](https://github.com/microsoft/vcpkg/files/11877310/config-x64-windows-dbg-CMakeCache.txt.log)
[config-x64-windows-out.log](https://github.com/microsoft/vcpkg/files/11877311/config-x64-windows-out.log)
[config-x64-windows-rel-CMakeCache.txt.log](https://github.com/microsoft/vcpkg/files/11877312/config-x64-windows-rel-CMakeCache.txt.log)

**Comment by dg0yt** [comment]
Fixed by #32227.

**Comment by FrankXie05** [comment]
This issue has been fixed by https://github.com/microsoft/vcpkg/pull/32227
